crates/tools/msvc/src/main.rs

```diff
@@ -9,6 +9,7 @@ fn main() {
             "x86" => "i686_msvc",
             "x64" => "x86_64_msvc",
             "arm64" => "aarch64_msvc",
+            "arm64ec" => "arm64ec_msvc",
```

Comment:
Hmm have you seen any tooling that sets Platform=arm64ec? The last official MSVC workflow I read involved using arm64 and passing ARM64EC flags into cl/link. So we'll probably need to tweak the arm64 workflow here and plumb through a `/machine:xxx` argument to be passed into lib.exe (on line 42).

Reply:
Presumably this works with `tool_msvc` but we need lib.yml to be updated to confirm.

Reply:
Actually, it occurs to me that this is all moot since Arm64EC links against the Arm64 import libraries in VC and the Windows SDK, so we can also reuse our existing AArch64 target crate.

Reply:
That would be even better and avoid another target lib crate.